src/e2e-test/features/spanner/source/SpannerToSpanner_Additional.feature

```diff
@@ -0,0 +1,118 @@
+@Spanner_Source @SPANNER_TEST
```

Comment:
What does @SPANNER_TEST do to the feature, and why isn't this tag present in the stepdesign package?

Reply:
It creates a random Spanner instance,database and table with 7 entries at the source.
It is an existing hook whose code was not altered for this PR.

Reply:
Noted